Source/WebCore/animation/WebAnimationUtilities.cpp

```diff
@@ -107,6 +108,19 @@ static bool compareStyleOriginatedAnimationOwningElementPositionsInDocumentTreeO
     auto& bReferenceElement = b.element;
 
     if (&aReferenceElement == &bReferenceElement) {
+        if (isNamedViewTransitionPseudoElement(a.pseudoElementIdentifier) && isNamedViewTransitionPseudoElement(b.pseudoElementIdentifier)) {
+            RefPtr activeViewTransition = aReferenceElement.document().activeViewTransition();
+            if (a.pseudoElementIdentifier->nameArgument != b.pseudoElementIdentifier->nameArgument && activeViewTransition) {
```

Comment:
It's probably best to do the `name != name` as part of the first if condition, so that we only access the `Document` when we need it.

Also, can we `ASSERT(activeViewTransition)`? Seems like we're in a bad spot if we have view transition pseudo elements, but no way to determine an order for them.